### PR TITLE
removed openlogin adapter

### PR DESCRIPTION
**Pull request by ihsraham**
replaced openloginadapter with authadapter

## Motivation and Context

Jira Link:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project. (run lint)
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/features/wallet-aggregation.mdx [hunk 1]
```diff
@@ -59,7 +59,7 @@ The table below summarizes all the available adapters and their properties
 
 | Adapter            | AuthMode         | ChainNamespace    | Default | Package Name                                                                     |
 | ------------------ | ---------------- | ----------------- | ------- | -------------------------------------------------------------------------------- |
-| `openlogin`        | `WALLET`, `DAPP` | `EIP155`,`SOLANA` | `YES`   | [`@web3auth/openlogin-adapter`](/sdk/pnp/web/adapters/openlogin)                 |
+| `openlogin`        | `WALLET`, `DAPP` | `EIP155`,`SOLANA` | `YES`   | [`@web3auth/openlogin-adapter`](/sdk/pnp/web/adapters/auth)                      |
```
Comment (shahbaz17) [inline docs/features/wallet-aggregation.mdx:62]: Please update the adapter name here.

File: docs/sdk/pnp/web/modal/initialize.mdx [hunk 2]
```diff
@@ -13,13 +13,13 @@ import UIConfigWhitelabelExample from "@site/src/common/sdk/pnp/web/_ui-config-w
 import Web3AuthOptions from "@site/src/common/sdk/pnp/web/_web3auth-options.mdx";
 import ChainConfig from "@site/src/common/sdk/pnp/web/_chain-config.mdx";
 import ExternalWalletAdapters from "@site/src/common/sdk/pnp/web/_external-wallet-adapters.mdx";
-import OpenloginAdapterIntro from "@site/src/common/sdk/pnp/web/_openlogin-adapter-intro.mdx";
+import OpenloginAdapterIntro from "@site/src/common/sdk/pnp/web/_auth-adapter-intro.mdx";
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/modal/initialize.mdx:16]: `OpenloginAdapterIntro` to `AuthAdapterIntro`

File: docs/sdk/pnp/web/modal/mfa.mdx [hunk 3]
```diff
@@ -4,8 +4,8 @@ sidebar_label: Multi Factor Authentication
 description: "@web3auth/modal Multi Factor Authentication | Documentation - Web3Auth"
 ---
 
-import OpenloginAdapterSettings from "@site/src/common/sdk/pnp/web/_openlogin-adapter-settings.mdx";
-import OpenloginLoginSettings from "@site/src/common/sdk/pnp/web/_openlogin-login-settings.mdx";
+import OpenloginAdapterSettings from "@site/src/common/sdk/pnp/web/_auth-adapter-settings.mdx";
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/modal/mfa.mdx:7]: same as above

File: docs/sdk/pnp/web/modal/whitelabel.mdx [hunk 4]
```diff
@@ -7,8 +7,8 @@ description: "Whitelabel Web3Auth | Documentation - Web3Auth"
 import InitModalConfigExample from "@site/src/common/sdk/pnp/web/_init-modal-config-example.mdx";
 import WhiteLabelShowcase from "@site/src/common/sdk/pnp/_whitelabel_showcase.mdx";
 import ModalConfig from "@site/src/common/sdk/pnp/web/_modal-config.mdx";
-import OpenloginWhiteLabelConfig from "@site/src/common/sdk/pnp/web/_openlogin-whitelabel-config.mdx";
-import OpenloginWhiteLabelExample from "@site/src/common/sdk/pnp/web/_openlogin-whitelabel-example.mdx";
+import OpenloginWhiteLabelConfig from "@site/src/common/sdk/pnp/web/_auth-whitelabel-config.mdx";
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/modal/whitelabel.mdx:10]: here too

File: docs/sdk/pnp/web/modal/whitelabel.mdx [hunk 5]
```diff
@@ -38,7 +38,7 @@ Within the Web3Auth Plug and Play Modal SDK, whitelabeling can happen in 3 diffe
 - [**Openlogin Adapter**](#openlogin-adapter): The redirect screens while logging in and
   constructing the key, can be customised according to your common as well. For this, you need to
   pass on the `whiteLabel` configuration parameter to the
-  [`openlogin-adapter`](/sdk/pnp/web/adapters/openlogin).
+  [`openlogin-adapter`](/sdk/pnp/web/adapters/auth).
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/modal/whitelabel.mdx:41]: `[`openlogin-adapter`]` to `[`auth-adapter`]`

File: docs/sdk/pnp/web/modal/whitelabel.mdx [hunk 6]
```diff
@@ -110,12 +110,12 @@ the social logins and their visibility in the modal.
 Web3Auth's Social Logins and Email Login run using the OpenLogin Flow. The whole OpenLogin user
 experience can also be whitelabeled using OpenLogin Adapter settings. For this, you need to pass on
 the `whiteLabel` configuration parameter to the `adapterSettings` property of the
-[`openlogin-adapter`](/sdk/pnp/web/adapters/openlogin).
+[`openlogin-adapter`](/sdk/pnp/web/adapters/auth).
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/modal/whitelabel.mdx:113]: same as above

File: docs/sdk/pnp/web/no-modal/custom-authentication.mdx [hunk 7]
```diff
@@ -44,8 +44,8 @@ among other things.
 
 :::tip
 
-Checkout the [`openlogin-adapter`](/sdk/pnp/web/adapters/openlogin) SDK Reference for more details
-on different configurations you can pass for customizations.
+Checkout the [`openlogin-adapter`](/sdk/pnp/web/adapters/auth) SDK Reference for more details on
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/no-modal/custom-authentication.mdx:47]: please check once for all these instances

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
@ihsraham

**Comment by ihsraham** [comment]
Rebased to another branch with it's PR here: https://github.com/Web3Auth/web3auth-docs/pull/911

**Pull request closed without merging**